packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1756,4 +1758,105 @@ describe("DeleteAttributeUseCases", () => {
             expect(CoreDate.from(updatedPredecessor.deletionInfo!.deletionDate).isBetween(timeBeforeUpdate, timeAfterUpdate.add(1))).toBe(true);
         });
     });
+
+    describe(DeleteThirdPartyOwnedRelationshipAttributeAndNotifyPeerUseCase.name, () => {
+        let peerSharedRelationshipAttribute: LocalAttributeDTO;
+        let thirdPartyOwnedRelationshipAttribute: LocalAttributeDTO;
+        beforeEach(async () => {
+            peerSharedRelationshipAttribute = await executeFullCreateAndShareRelationshipAttributeFlow(services3, services1, {
+                content: {
+                    value: {
+                        "@type": "ProprietaryString",
+                        value: "My amazing string",
+                        title: "Nothing is this amazing"
+                    },
+                    key: "amazing",
+                    confidentiality: RelationshipAttributeConfidentiality.Public
+                }
+            });
+
+            const requestParams = {
+                peer: services1.address,
+                content: {
+                    items: [
+                        ReadAttributeRequestItem.from({
+                            query: ThirdPartyRelationshipAttributeQuery.from({ key: "amazing", owner: services3.address, thirdParty: [services3.address] }),
+                            mustBeAccepted: true
+                        }).toJSON()
+                    ]
+                }
+            };
+
+            thirdPartyOwnedRelationshipAttribute = await executeFullRequestAndShareThirdPartyRelationshipAttributeFlow(
+                services1,
+                services2,
+                requestParams,
+                peerSharedRelationshipAttribute.id
+            );
+        });
+
+        test("should delete a third party owned RelationshipAttribute as the sender", async () => {
```

Comment:
IMHO: Normally, we refer to the Sender of a Request as the Sender and the Recipient of a Request as the Recipient. Here the designations are used the other way round, because we refer to the Attributes instead of the Request used, which makes sense because it is about the Attributes and not the Request here. However, I'd reformulate to "should delete a third party owned RelationshipAttribute as the sender **of it**" or try to find a synonym for the Sender to avoid confusion. 

Same for the tests below (with Recipient or Sender).